src/go/rpk/Makefile

```diff
@@ -50,4 +50,27 @@ help: ## Show this help.
 	@awk 'BEGIN {FS = ":.*?## "} { \
 		if (/^[a-zA-Z_-]+:.*?##.*$$/) {printf "    ${YELLOW}%-20s${GREEN}%s${RESET}\n", $$1, $$2} \
 		else if (/^## .*$$/) {printf "  ${CYAN}%s${RESET}\n", substr($$1,4)} \
-		}' $(MAKEFILE_LIST)
\ No newline at end of file
+		}' $(MAKEFILE_LIST)
+
+# Run crlfmt against code
+fmt: crlfmt
```

Comment:
shouldn't we be pushing this to task? 

cc: @ivotron can we port this makefile to task, so make proxies to task.

Reply:
what is the advantage of doing this in task since it's mainly for local development?

Reply:
it's the same code (task == shell) and we can have one source of truth for local and for CI, so unifying tasks, but i'll let ivo comment on it too.

Reply:
yes, that's in the backlog. I'll try to find time next week. In the meantime I think we could merge this